README.md

```diff
@@ -24,6 +24,36 @@ file after installation and remove `akida_pcie` line.
 
 ***After a kernel update you need to run the install script again.***
 
+## Enable CMA in the kernel
+
+Some systems, e.g.: Ubuntu on x86_64, do not come with CMA (contiguous memory
+allocation) support enabled in the kernel. This is required to use AKD1500
+devices through PCIe if you want to use larger amounts of memory to program big
+models or make full usage of the pipeline. To build the kernel packages that
+include such support you can run the dedicated script, tailored for the Ubuntu
+distribution:
+
+```
+./build_kernel_w_cma.sh
+```
+
+The script will create the packages and explain how to install and boot on the
+new kernel.
+
+If you want to go back to an old kernel that does not contain the CMA feature,
+you can grep the installed kernels that were configured in grub:
+
+```
+grep 'menuentry \|submenu ' /boot/grub/grub.cfg | cut -f2 -d "'"
+```
+
+And then modify the `/etc/default/grub` file from: `GRUB_DEFAULT=0`
+to: `GRUB_DEFAULT=2` if you want the third kernel choice in the obtained list,
+etc.
+
+Source: https://unix.stackexchange.com/questions/432393/downgrade-linux-kernel-without-grub
+
```

Comment:
We could enforce the massage "After a kernel update you need to run the install script again" here again :)